pgoapi/pgoapi.py

```diff
@@ -218,8 +220,15 @@ def heartbeat(self):
                 res['responses']['lng'] = self._posf[1]
                 f.write(json.dumps(res['responses'], indent=2))
 
+            inventory_items = res['responses']['GET_INVENTORY']['inventory_delta']['inventory_items']
+            self.inventory = Player_Inventory(inventory_items)
+
+            for inventory_item in inventory_items:
```

Comment:
can we make a file called player.py with Player class and fill it up with this info and create a __str__ method like the inventory class to encapsulate this data?